### PR TITLE
fix(bounty): fixes the copy button to correctly copy the bounty url o…

### PR DESCRIPTION
**Pull request by dylanbathurst**
…n both web and mobile, also fixes a case where bounty url was showing undefined, cleans up some unused code

## Describe your changes
Fixes the copy to clipboard button text by setting the proper bounty path `/bounty/${id}` and cleaning up some code that wasn't being used anywhere.
https://www.loom.com/share/d8e612468b8040c3aa85516b41690d4a

## Issue ticket number and link
https://community.sphinx.chat/bounty/656

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [x] I have tested on a mobile device
- [x] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

File: frontend/app/src/config/host.ts [hunk 1]
```diff
@@ -3,7 +3,8 @@ const externalDockerHosts = ['localhost:23007', 'localhost:23000'];
 
 export function getHost(): string {
   const host = window.location.host.includes('localhost') ? 'localhost:5002' : window.location.host;
-  return host;
+  // return host;
+  return 'people.sphinx.chat'
```
Comment (kevkevinpal) [inline frontend/app/src/config/host.ts:7]: might want to remove this from the diff

File: frontend/app/src/config/ModeDispatcher.tsx [hunk 2]
```diff
@@ -7,7 +7,7 @@ export enum AppMode {
 }
 
 const hosts: { [k: string]: AppMode } = {
-  'localhost:3000': AppMode.TRIBES,
+  'localhost:3000': AppMode.COMMUNITY,
```
Comment (kevkevinpal) [inline frontend/app/src/config/ModeDispatcher.tsx:10]: might want to remove this from the diff

File: frontend/app/src/config/host.ts (hunk 1 above)
Comment (dylanbathurst) [reply frontend/app/src/config/host.ts]: Yikes, my bad. Fixe.

File: frontend/app/src/config/ModeDispatcher.tsx (hunk 2 above)
Comment (dylanbathurst) [reply frontend/app/src/config/ModeDispatcher.tsx]: Fixed

File: frontend/app/src/config/host.ts (hunk 1 above)
Comment (dylanbathurst) [reply frontend/app/src/config/host.ts]: @kevkevinpal

**Comment by elraphty** [comment]
@kevkevinpal, It looks good to me.

**Comment by kevkevinpal** [comment]
looking good to me will merge when the gitactions are passing!

**Pull request merged**